prow/jobs/kyma/kyma-integration.yaml

```diff
@@ -228,8 +233,8 @@ periodics:
   extra_refs:
   - <<: *test_infra_ref
     base_ref: master
-  - org: kyma-project
-    repo: kyma
+  - <<: *kyma_ref
+    base_ref: master
     path_alias: github.com/kyma-project/kyma
```

Comment:
`path_alias` is duplicated. You can remove this one.

Reply:
Good catch, thanks